src/model/HeatExchangerAirToAirSensibleAndLatent.cpp

```diff
@@ -744,5 +804,196 @@ namespace model {
+    if (boost::optional<TableLookup> sensibleEffectivenessofCoolingAirFlowCurve_ =
+          sensibleEffectivenessofCoolingAirFlowCurve()->optionalCast<TableLookup>()) {
+      return sensibleEffectivenessofCoolingAirFlowCurve_->outputValues()[0];
+    } else {
+      return getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->sensibleEffectivenessat100CoolingAirFlow();
+    }
+  }
+
+  double HeatExchangerAirToAirSensibleAndLatent::latentEffectivenessat75CoolingAirFlow() const {
+    DEPRECATED_AT_MSG(3, 8, 0, "As of EnergyPlus 24.1.0, this property ...");
+    if (boost::optional<TableLookup> latentEffectivenessofCoolingAirFlowCurve_ =
+          latentEffectivenessofCoolingAirFlowCurve()->optionalCast<TableLookup>()) {
+      return latentEffectivenessofCoolingAirFlowCurve_->outputValues()[0];
+    } else {
+      return getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->latentEffectivenessat100CoolingAirFlow();
+    }
+  }
+
+  bool HeatExchangerAirToAirSensibleAndLatent::setSensibleEffectivenessat75HeatingAirFlow(double sensibleEffectivenessat75HeatingAirFlow) {
+    DEPRECATED_AT_MSG(3, 8, 0, "As of EnergyPlus 24.1.0, this property ...");
+    if (boost::optional<TableLookup> sensibleEffectivenessofHeatingAirFlowCurve_ =
+          sensibleEffectivenessofHeatingAirFlowCurve()->optionalCast<TableLookup>()) {
+      double sensibleEffectivenessat100HeatingAirFlow =
+        getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->sensibleEffectivenessat100HeatingAirFlow();
+      sensibleEffectivenessofHeatingAirFlowCurve_->setOutputValues(
+        {sensibleEffectivenessat75HeatingAirFlow, sensibleEffectivenessat100HeatingAirFlow});
+      return setSensibleEffectivenessofHeatingAirFlowCurve(sensibleEffectivenessofHeatingAirFlowCurve_.get());
+    } else if (!sensibleEffectivenessofHeatingAirFlowCurve()) {
+      double sensibleEffectivenessat100HeatingAirFlow =
+        getImpl<detail::HeatExchangerAirToAirSensibleAndLatent_Impl>()->sensibleEffectivenessat100HeatingAirFlow();
+
+      TableIndependentVariable var(model());
+      var.setInterpolationMethod("Linear");
+      var.setExtrapolationMethod("Linear");
+      var.setMinimumValue(0.0);
+      var.setMaximumValue(10.0);
+      var.setUnitType("Dimensionless");
+      var.addValue(0.75);
+      var.addValue(1.0);
+
+      TableLookup tableLookup(model());
+      tableLookup.setNormalizationMethod("DivisorOnly");
+      tableLookup.setNormalizationDivisor(sensibleEffectivenessat100HeatingAirFlow);
+      tableLookup.setMinimumOutput(0.0);
+      tableLookup.setMaximumOutput(10.0);
+      tableLookup.setOutputUnitType("Dimensionless");
+      tableLookup.addOutputValue(sensibleEffectivenessat75HeatingAirFlow);
+      tableLookup.addOutputValue(sensibleEffectivenessat100HeatingAirFlow);
+      tableLookup.addIndependentVariable(var);
+
+      return setSensibleEffectivenessofHeatingAirFlowCurve(tableLookup);
```

Comment:
No ambiguity in this case: if the curve isnt there, we **can** make a table lookup indeed.